src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java

```diff
@@ -134,6 +130,7 @@
         oneLineSummary = "Calculate genotype posterior probabilities given panel data",
         programGroup = VariantProgramGroup.class
 )
+@DocumentedFeature
```

Comment:
Are we still doing the categories in GATK4?  This was in Variant Discovery in GATK3.